v2/decoder_activation.go

```diff
@@ -56,6 +62,9 @@ func (oa v1ActivationClaims) Migrate() (*ActivationClaims, error) {
 }
 
 func (oa v1ActivationClaims) migrateV1() (*ActivationClaims, error) {
+	if oa.Max != NoLimit || oa.Payload != NoLimit || oa.Src != "" || len(oa.Times) != 0 {
+		return nil, fmt.Errorf("jwtV1 activation: deprecated Max/Payload/Src/Times in jwtV2")
```

Comment:
so if we find a deprecated field, we cannot upgrade the JWT? Just wondering what the upgrade process is if you already upgraded all your tooling. IE ngs needs to internally migrate and this gets in the way.

Reply:
I don't believe these fields where ever specified or checked, which is why returning an error seems appropriate